common/quotas/global/collection/internal/fallback_test.go

```diff
@@ -19,7 +19,7 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
-package internal_test
+package internal
```

Comment:
mostly to share the allow-limiter.  in principle I still like having this external because it doesn't need to be internal, but meh.  it's rather minor.